frontend/packages/core/src/button.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import type { Meta } from "@storybook/react";
+
+import type { ButtonProps } from './button'
+
+import { Button } from './button';
+
+export default {
+  title: 'Core/Button/Button',
+  component: Button,
+
```

Comment:
can we add an [argType](https://storybook.js.org/docs/react/api/argtypes) here for the onClick handler

Reply:
added!